src/com/company/Parse.java

```diff
@@ -0,0 +1,120 @@
+        repo = new HashMap<>();
+
+        translationUnit = getIASTTranslationUnit(code.toCharArray());
+        IASTNode node = translationUnit;
+
+        saveNodes(node);
+    }
+
+    /**Accesses all the nodes of the tree*/
+    private void saveNodes(IASTNode node) throws Exception {
+        IASTNode[] children = node.getChildren();
+        for (IASTNode child :children) {
+            if (child instanceof CPPASTSimpleDeclaration) {
+
+                /**Setup*/
+                CPPASTSimpleDeclaration declaration = (CPPASTSimpleDeclaration) child;
+                CPPASTNamedTypeSpecifier specifier = (CPPASTNamedTypeSpecifier) declaration.getDeclSpecifier();
+                CPPASTDeclarator declarator = (CPPASTDeclarator) declaration.getDeclarators()[0];
+                CPPASTEqualsInitializer init = (CPPASTEqualsInitializer) declarator.getInitializer();
+
+                if (init == null) {
+                    continue;
+                }
+
+                if (declarator == null) {
+                    continue;
+                }
+
+                if (specifier == null) {
+                    continue;
+                }
+
+                if (declaration == null) {
+                    continue;
+                }
+
+                CPPASTLiteralExpression value = (CPPASTLiteralExpression) init.getInitializerClause(); //get value
+                CPPASTName name = (CPPASTName) declarator.getName(); //get name
+                IToken typeQualifier = specifier.getSyntax(); //get typeQualifer type (iToken)
+                CPPASTName type = (CPPASTName) specifier.getName(); //get type
+
+                Contents contents = new Contents(name, value, typeQualifier, type);
+                repo.put(name.getRawSignature(), contents);
+            }
+        }
+    }
+
+    public Contents get(String key) throws IllegalAccessError{
+        if (repo.get(key) == null) {
+            throw new IllegalAccessError("The name does not exist!");
+        }
+        return repo.get(key);
+    }
+
+    public ArrayList<Contents> getByType(String key) {
+        return findList(key, 0);
+    }
+
+    public ArrayList<Contents> getTypeQualifier(String key) {
+        return findList(key, 1);
+    }
+
+    /**Makes and return a list of contents*/
+    private ArrayList<Contents> findList (String key, int i) {
+        ArrayList<Contents> list = new ArrayList<>();
+        for (String con : repo.keySet()) {
+            Contents contents = get(con);
+            if (i == 0) {
+                if (contents.type.equals(key)) {
+                    list.add(contents);
+                }
+            }
+            else {
+                if (contents.typeQualifer.equals(key)) {
+                    list.add(contents);
+                }
+            }
+        }
+        if (list.isEmpty()) {
+            throw new IllegalArgumentException("No contents[s] were found for given key.");
+        }
+        return list;
+    }
```

Comment:
What's the use case for this? If it's just to enable testing, consider moving this into the test code as a utility function.